create_items.rb

```diff
@@ -0,0 +1,109 @@
+require_relative 'game'
+require_relative 'author'
+require_relative 'musicalbum'
+require_relative 'book'
+
+module NewItem
+  def initialize
+    @item_options = '0'
+  end
+
+  def item_options
+    puts 'which item would you like to create?'
+    puts '1. Add a Book'
+    puts '2. Add a Music Album'
+    puts '3. Add a Game'
+  end
+
+  def add_selected_item
+    # @item_options = gets.chomp
```

Comment:
**You all did great job. Well implemented methods and tests.**

- [ ] However, I think this one was just out of your attention due to some reasont. Do not worry, it's not the end of the world. You are free to fix this up. If this commented code is not necessary action in this project. Please, remove it. Thank you!

Reply:
### Thanks We have resolve it!